### ISSUE TITLE
Load SQL dumps from On-Prem Oracle DB into AWS RDS

### ISSUE THREAD

**Issue by davidkwartler** [US-EPA-CAMD/PI-1#101]
- [x] Obtain SQL dumps from on-prem Oracle database
- [ ] Load data into CVP AWS Oracle RDS

A/C:
Data is loaded into CVP AWS Oracle RDS and available to the developers

**Comment by davidkwartler** [comment]
Blocked as of 8/27
Waiting on data dumps

**Comment by davidkwartler** [comment]
Blocked as of 8/31
Jason submitted ticket for access to data dumps, working with Mike

**Comment by davidkwartler** [comment]
Blocked as of 9/1
Mike got in touch with EPA folks

**Comment by david-cvp** [comment]
9/10 Jason sent ticket to Cloud.gov, received response that they dont support S3 integration for RDS data pump import

Will proceed with CVP AWS

**Comment by david-cvp** [comment]
As of 9/14, CVP AWS Oracle RDS is set up and ready to load data
Running into one problem with an error

**Comment by david-cvp** [comment]
As of 9/15, this issue was moved to Sprint 1.3. 
Jason is currently working to load the data into our CVP AWS Oracle RDS database from an S3 bucket. He has run into some user permission issues and is working to complete the data import so he may provide the developers with access to the database ASAP.

**Comment by jwhitehead77** [comment]
This has been over come by events and no longer needed
